test/checkpoint/test_checkpoint.c

```diff
@@ -327,9 +339,39 @@ main(int argc, char *argv[])
     /* Ensure that cleanup is done on error. */
     (void)wt_shutdown();
     free(g.cookies);
+    testutil_cleanup(&g.opts);
+
     return (g.status);
 }
 
```

Comment:
Comment needed here, example - 
/*
 * wt_connect --
 *     Configure the WiredTiger connection.
 */